tools/ui-components/src/button/button.tsx

```diff
@@ -120,7 +132,6 @@ export const Button = React.forwardRef<HTMLButtonElement, ButtonProps>(
     const handleClick = useCallback(
       (event: React.MouseEvent<HTMLButtonElement>) => {
         const ariaDisabled = event.currentTarget.getAttribute('aria-disabled');
-
```

Comment:
Hope you don't mind keeping this empty line. I find the code a bit easier to read with it 🙂

Reply:
For sure! Do I need to edit anything in my commit to keep the empty line? (new to working with open-source).

Reply:
GitHub doesn't allow suggestions on deleted lines, so you would need to add the line back locally, then commit and push the change.

Let me know if you need help with this.

Reply:
Just commited with the line added back in, is this right?

Reply:
Yup! The removal is reverted 👍